Tests/ETCDTests/ETCDGetSetTests.swift

```diff
@@ -0,0 +1,43 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-etcd-client-gsoc open source project
+//
+// Copyright (c) 2024 Apple Inc. and the swift-etcd-client-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-etcd-client-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import XCTest
+import NIO
+@testable import ETCD
+
+final class EtcdClientTests: XCTestCase {
+    var eventLoopGroup: EventLoopGroup!
+    var etcdClient: EtcdClient!
+
+    override func setUp() async throws {
+        eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
Let's just use the singleton here
```suggestion
        eventLoopGroup = MultiThreadedEventLoopGroup.singleton
```